matgen/generate_type_ij.hh

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2017-2022, University of Tennessee. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#ifndef SLATE_GENERATE_TYPE_IJ_HH
+#define SLATE_GENERATE_TYPE_IJ_HH
+
+
+#include "slate/slate.hh"
+#include "test.hh"
+#include "print_matrix.hh"
+
+#include <exception>
+#include <string>
+#include <vector>
+#include <limits>
+#include <complex>
+#include <chrono>
+
+#include <cmath>
+#include <cstdio>
+#include <cstdlib>
+#include <utility>
+
+#include "matrix_params.hh"
+#include "generate_matrix.hh"
+#include "random.hh"
+
+
+namespace slate {
+
+template <typename scalar_t>
+void generate_ij( slate::Matrix<scalar_t>& A,
```

Comment:
We can implement `ij` with `set` and the following lambda.
```
[s] (int64_t i, int64_t j) {return i + j * s;}
```

Reply:
Implemented. Thank you for the comment.